### ISSUE TITLE
FreeType macOS issue: Bleeding

### ISSUE THREAD

**Issue by danoli3** [openframeworks/apothecary#404]
Currentlying getting some linking issues with latest libs from 2 days ago.

I did do a force clean build on this so no caches.
```

Showing All Messages
Undefined symbol: _hb_blob_create

Undefined symbol: _hb_blob_destroy

Undefined symbol: _hb_buffer_add_utf8

Undefined symbol: _hb_buffer_clear_contents

Undefined symbol: _hb_buffer_create

Undefined symbol: _hb_buffer_destroy

Undefined symbol: _hb_buffer_get_glyph_infos

Undefined symbol: _hb_buffer_get_glyph_positions

Undefined symbol: _hb_buffer_get_length

Undefined symbol: _hb_buffer_guess_segment_properties

Undefined symbol: _hb_face_create

Undefined symbol: _hb_face_create_for_tables

Undefined symbol: _hb_face_destroy

Undefined symbol: _hb_face_set_index

Undefined symbol: _hb_face_set_upem

Undefined symbol: _hb_font_create

Undefined symbol: _hb_font_destroy

Undefined symbol: _hb_font_get_face

Undefined symbol: _hb_font_set_scale

Undefined symbol: _hb_ot_layout_collect_lookups

Undefined symbol: _hb_ot_layout_lookup_collect_glyphs

Undefined symbol: _hb_ot_layout_lookup_would_substitute

Undefined symbol: _hb_ot_tags_from_script_and_language

Undefined symbol: _hb_set_create

Undefined symbol: _hb_set_destroy

Undefined symbol: _hb_set_is_empty

Undefined symbol: _hb_set_next

Undefined symbol: _hb_set_subtract

Undefined symbol: _hb_shape

Linker command failed with exit code 1 (use -v to see invocation)


```

**Comment by danoli3** [comment]
Also PR to FreeType to fix ARM64EC was merged on Upstream/FreeType: https://gitlab.freedesktop.org/freetype/freetype/-/merge_requests/334?commit_id=75a9fa61a8822ac0b0746b6bb70a97809280fc7b

**Comment by danoli3** [comment]
> Also PR to FreeType to fix ARM64EC was merged on Upstream/FreeType: https://gitlab.freedesktop.org/freetype/freetype/-/merge_requests/334?commit_id=75a9fa61a8822ac0b0746b6bb70a97809280fc7b

Unrelated ** hehe

**Comment by danoli3** [comment]
HB must be related to HarfBuzz

**Comment by danoli3** [comment]
Was fixed with FreeImage 3.19.9
